keep/api/core/db.py

```
@@ -4556,9 +4513,8 @@ def get_activity_report(
             .filter(Workflow.creation_time >= last_24_hours)
             .count()
         )
-        activity_report["last_24_hours_workflows_executed"] = (
-            session.query(WorkflowExecution)
-            .filter(WorkflowExecution.started >= last_24_hours).count()
+        activity_report["last_24_hours_workflows_executed"] = (session.query(WorkflowExecution).filter(
```

Ruff (0.8.0)

`WorkflowExecution` may be undefined, or defined from star imports

(F405)